cpp/src/protocol/Base64Util.cpp

```diff
@@ -179,8 +179,9 @@ velox::VectorPtr readVariableWidthBlock(const velox::TypePtr& type, ByteStream&
     // TBD: Performance optimization
     for (auto i = 0; i < positionCount; ++i) {
       if (velox::bits::isBitNull(nulls->as<uint64_t>(), i)) {
-        rawBuffer[i] = velox::StringView(rawString + offset, rawLens[i]);
-        offset += rawLens[i];
+        // rawBuffer[i] = velox::StringView(rawString + offset, rawLens[i]);
+        // offset += rawLens[i];
+        rawBuffer[i] = velox::StringView("", 0);
```

Comment:
@bigPYJ1151 PTAL

I logged the rawString here, and it prints various and wrong values -- the rawLens[i] may even be a negative number，which may cause exception creating StringView.
The simplest SQL could be, say,
```sql
select i_category, i_class,
  rank() over (
      partition by grouping(i_category)+grouping(i_class),
      case when false then i_category end
      ) as rank_within_parent
from
   item
group by rollup(i_category,i_class)
order by
  rank_within_parent;
```
Probably tricky solution currently, do you have a better method?

Reply:
Seems the condition should be ```!velox::bits::isBitNull(nulls->as<uint64_t>(), i)```